includes/Modules/Analytics.php

```diff
@@ -126,6 +126,86 @@ function () {
 			},
 			0
 		);
+
+		add_action(
+			'admin_init',
+			function() {
+				$this->handle_provisioning_callback();
+			}
+		);
+	}
+
+	/**
+	 * Handles the provisioning callback after the user completes the terms of service.
+	 *
+	 * @since 1.0.0
+	 */
+	private function handle_provisioning_callback() {
```

Comment:
This is not urgent, but since the version number needs to be changed anyway would be good to do: This method should rather be `protected` like similar ones and be moved a bit further down in the class, together with the other protected methods that are hooked in in the `register()` method.